include/fmt/core.h

```diff
@@ -2029,14 +2029,20 @@ template <typename Context> class basic_format_args {
 // between clang and gcc on ARM (#1919).
 using format_args = basic_format_args<format_context>;
 
-// We cannot use enum classes as bit fields because of a gcc bug
+// We cannot use enum classes as bit fields because of a gcc bug before 9.3
```

Comment:
This seems to be about  underlying types rather than enum classes because we don't use the former in both branches below.

Reply:
I'm not so sure they are different bugs in fact.  But in any case I've improved both the comments in the code and the commit message.

Reply:
Thanks!